docs/Releasing.md

```diff
@@ -1,3 +1,15 @@
+# Building and Debuging
+
+Like `rustc` and `cargo`, `sccache` uses a separate `env_logger` environment
+variable to configure the logging. For sccache this is called `SCCACHE_LOG`.
```

Comment:
I don't think a section on "building and debugging" belongs in documentation about releasing `sccache`.  You probably want to update https://github.com/mozilla/sccache#debugging instead.

Reply:
Ah yes totally - will make this change.

Reply:
Done.